guest/s2ebios/tests/tests64.asm

```diff
@@ -53,31 +53,31 @@ test_memory_rw_new_page:
 
     ; 1 byte rw
     mov al, 0xdf
-    lea rcx, 0x20000
+    mov rcx, 0x20000
```

Comment:
nasm on my system doesn't throw any error on this. I am curious what's on yours.

Reply:
It throws error: invalid combination of opcode and operands